### PR TITLE
Add kubeconform template for schema validation

### PR DESCRIPTION
**Pull request by janisz**
This PR introduces a new kubeconform template that validates k8s resources against their schemas using the [kubeconform](https://github.com/yannh/kubeconform) library.

Fixes:
- #36 

🤖 Generated with [Claude Code](https://claude.ai/code)

### CODE REVIEW

File: pkg/templates/kubeconform/template_test.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+		},
+		{
+			name:  "multiple elements",
+			input: []string{"pod", "service", "deployment"},
+			expected: map[string]struct{}{
+				"pod":        {},
+				"service":    {},
+				"deployment": {},
+			},
+		},
```
Comment (rhybrillou) [inline pkg/templates/kubeconform/template_test.go]: Would it make sense to add a test for input array with duplicates ?

**Review by rhybrillou — APPROVED** [review]
